<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>864</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QProgressBar{
	border:0.1px solid grey;
	background-color:rgb(74,75,83)
}
QProgressBar::chunk{
	background-color:rgb(0,185,84);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1200</width>
      <height>864</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/bg3.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>235</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>主机温度</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>392</x>
      <y>237</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>主轴转速</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>237</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>副轴转速</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>236</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>主机时间</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>324</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>1号压力仓</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>56</x>
      <y>413</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>2号压力仓</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>502</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>3号压力仓</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>590</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>4号压力仓</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>680</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>5号压力仓</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>770</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>1号工作杠油量</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>770</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>2号工作杠油量</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>770</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>3号工作杠油量</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>350</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>机械臂压力</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_temperature">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>---</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_zhu_zhou">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>270</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>---</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_fu_zhou">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>270</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>---</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_yl_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>800</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>000 L</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_yl_1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>800</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>000 L</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_yl_3">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>800</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>000 L</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_yl_1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>330</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>--- Pa</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_yl_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>410</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>--- Pa</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_yl_4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>590</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>--- Pa</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_time">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>270</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>---</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_jxb">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>350</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>--- Pa</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_yl_5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>680</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>--- Pa</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_yl_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>500</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>--- Pa</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="pb_cang_1">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>357</y>
      <width>208</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="minimum">
     <number>500</number>
    </property>
    <property name="maximum">
     <number>1200</number>
    </property>
    <property name="value">
     <number>800</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="pb_cang_2">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>444</y>
      <width>208</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>500</number>
    </property>
    <property name="maximum">
     <number>1200</number>
    </property>
    <property name="value">
     <number>800</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="pb_cang_3">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>534</y>
      <width>208</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>500</number>
    </property>
    <property name="maximum">
     <number>1200</number>
    </property>
    <property name="value">
     <number>800</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="pb_cang_4">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>623</y>
      <width>208</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>500</number>
    </property>
    <property name="maximum">
     <number>1200</number>
    </property>
    <property name="value">
     <number>800</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="pb_cang_5">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>711</y>
      <width>208</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>500</number>
    </property>
    <property name="maximum">
     <number>1200</number>
    </property>
    <property name="value">
     <number>800</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="pb_jxb_6">
    <property name="geometry">
     <rect>
      <x>967</x>
      <y>381</y>
      <width>208</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="minimum">
     <number>500</number>
    </property>
    <property name="maximum">
     <number>1200</number>
    </property>
    <property name="value">
     <number>800</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>360</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>折线图</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
